controllers/get-topics.controller.js

```diff
@@ -0,0 +1,8 @@
+const {selectTopics} = require('../models/select-topics.model.js')
+
+exports.getTopics = (request, response) => {
+    selectTopics()
+        .then((topicsData) => {
+            response.status(200).send(topicsData)
+        })
```

Comment:
Nice, but just to be on the safe side you can and should still add a .catch(...) here. Just in case something unexpected goes wrong we can still catch it and try to deal with it in some way.